app/services/performance_dashboard_service.rb

```diff
@@ -84,4 +84,24 @@ def users_not_active
   def users_active_30_days
     number_with_delimiter(@response["publish"]["users"]["recent_active_users"])
   end
+
+  def published_courses
+    number_with_delimiter(@response["rollover"]["total"]["published_courses"])
```

Comment:
We should add an accessor for `rollover_total` to remove the duplication in these methods. e.g.

```
def rollover_total
  @response["rollover"]["total"]
end

Reply:
Thanks, all fixed